index.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'markovchain';
```

Comment:
Pablo might have input but is there a better way to do this other than just dumping this declaration in a file in the root directory?

Reply:
Did a declare not work here? https://github.com/heavenly-gumption/stinky-bot/blob/master/src/bot.ts

A quick look at this package and its alternative show me no types in the repositories, nor DefinitelyTyped. https://www.typescriptlang.org/dt/search?search=markov

Not sure what the best approach here is, to be quite honest. These packages are semi-ancient at this point, and not sure what kind of vulnerabilities they may have that we will eventually (not) deal with due to GitHub's alerts.

I'm fine with a declare somewhere else, but I agree with Chu here. Is there an alternative you'd prefer Nagn?

Reply:
Yeah this markovchain library is ancient. I briefly tried porting it to markov-strings, which I got to work, but that one doesn't actually have the ability to seed a markov sentence with an arbitrary word, and the meme content was poor.

It might be a good exercise to try to re-implement our own from scratch. I'll see if I can move the declare to bot.ts

Reply:
It looks like that at the very least vscode complains when I move the declare to the bot.ts, so it'll have to be in its own file, acroding to this
https://stackoverflow.com/questions/52481655/module-augmentation-defining-types-over-non-typed-files

So I moved it to a file in /src